src/helpers/pivot/spreadsheet_pivot/spreadsheet_pivot.ts

```diff
@@ -224,10 +225,20 @@ export class SpreadsheetPivot implements Pivot<SpreadsheetPivotRuntimeDefinition
     if (!operator) {
       throw new Error(`Aggregator ${aggregator} does not exist`);
     }
-    return {
-      value: values.length ? operator.fn([values], this.getters.getLocale()) : "",
-      format: operator.format(values[0]),
-    };
+
+    try {
+      return {
+        value: values.length ? operator.fn([values], this.getters.getLocale()) : "",
+        format: operator.format(values[0]),
+      };
+    } catch (e) {
+      const errorFPayload = {
```

Comment:
I think we should just proces the "expected" evaluation errors. Programatic errors SHOULD make the code crash. Id add a filtering with `isEvaluationError`. WDYT?

Reply:
I'm not against the idea, but I copied the behaviour of the evaluation (`evaluator.ts computeCell`), so it might be an intended design choice ?